lab-davide/data/notes/0fbe7160-43cb-11e6-9936-1f4e9c8af7f6

```diff
@@ -0,0 +1 @@
+{"id":"0fbe7160-43cb-11e6-9936-1f4e9c8af7f6","content":"test note","timestamp":"2016-07-06T22:43:30.295Z"}
```

Comment:
You should add your notes folder to your .gitignore so that these don't show up in your pull requests.